README.md

```diff
@@ -119,81 +119,18 @@ There is no capability to run an https server at this point in time, and no auth
 addresses).  You could also put a reverse proxy in front of the service.  Documentation for the endpoints can be found
 under HTTP.md
 
-Configuring backends and cloud providers
-----------------------------------------
-Backends and cloud providers are configured using `toml`, `json` or `yaml` configuration file
-passed via the `--config-path` flag. For all configuration options see source code of the backends you
-are interested in. A cloudprovider should not be used on the aggregation server when forwarding data to
-it, as the source IP address is not propagated.  A cloudprovider can be used on the forwarder host, however.
-Configuration file might look like this:
-```
-[graphite]
-	address = "192.168.99.100:2003"
-
-[datadog]
-	api_key = "my-secret-key" # Datadog API key required.
-
-[statsdaemon]
-	address = "docker.local:8125"
-	disable_tags = false
-
-[aws]
-	max_retries = 4
-
-[newrelic]
-	address = "http://localhost:8001/v1/data"
-	event-type = "GoStatsD"
-	#see full configuration options further below
-```
-
-New Relic Backend
------------------------------
-Supports two routes for flushing metrics to New Relic.
-- Directly to the Insights Collector - [Insights Event API](https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/send-custom-events-event-api)
-- Via the Infrastructure Agent's inbuilt HTTP Server
-
-### [New Relic Insights Event API](https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/send-custom-events-event-api)
-Sending directly to the Event API alleviates the requirement of needing to have the New Relic Infrastructure Agent. Therefore you can run this from nearly anywhere for maximum flexibility. This also becomes a shorter data path with less resource requirements becoming a simpler setup.
-
-To use this method, create an Insert API Key from here: https://insights.newrelic.com/accounts/YOUR_ACCOUNT_ID/manage/api_keys
+Configuring backends
+--------------------
+Refer to [BACKENDS.md] for configuration options for the backends
```

Comment:
Is this link meant to be clickable? Because when previewing it I see the text `[BACKENDS.md]`

Reply:
I'm not too sure, actually.  It's the style used throughout the file when referencing local files, so I just followed it.